core-to-plc/src/Language/Plutus/CoreToPLC/Plugin.hs

```diff
@@ -3,7 +3,7 @@
 {-# LANGUAGE OverloadedStrings          #-}
 {-# LANGUAGE TemplateHaskell            #-}
 {-# LANGUAGE ViewPatterns               #-}
-module Language.Plutus.CoreToPLC.Plugin (PlcCode, getSerializedCode, applyPlc, getAst, plugin, plc) where
+module Language.Plutus.CoreToPLC.Plugin (PlcCode(PlcCode), getSerializedCode, applyPlc, getAst, plugin, plc) where
```

Comment:
As it says in the module, this type is *deliberately* abstract, can we not expose the constructor, please? I think you should just be able to put the typeclass instance in here, which also avoids orphans.